fiona/fio/calc.py

```diff
@@ -0,0 +1,79 @@
+import json
+import logging
+import math
+from munch import munchify
+
+import click
+from cligj import use_rs_opt
+
+from .helpers import obj_gen
+
+
+@click.command(short_help="Calculate GeoJSON property by Python expression")
+@click.argument('property_name')
+@click.argument('expression')
+@click.option('--overwrite', '-o', is_flag=True, default=False,
```

Comment:
@perrygeo I wonder if we shouldn't reserve `-o` for future use?  It's a pretty standard flag for specifying output, and while Fiona currently almost exclusively writes to `stdout`, it will likely create problems if we switch to writing directly to disk.  The CLI is pretty consistent as-is, so I don't feel too strongly about this.